### PR TITLE
[IMP] *: remove legacy services and env

### PR DESCRIPTION
**Pull request by mcm-odoo**

### CODE REVIEW

**Comment by ged-odoo** [comment]
robodoo override=ci/security

File: addons/web/static/src/start.js [hunk 1]
```diff
@@ -31,10 +29,10 @@ export async function startWebClient(Webclient) {
     const env = makeEnv();
     await startServices(env);
 
+    Component.env = env;
```
Comment (Polymorphe57) [inline addons/web/static/src/start.js:32]: add comment "FOR LEGACY"

File: addons/website_event/static/src/js/register_toaster_widget.js [hunk 2]
```diff
@@ -16,9 +20,8 @@ publicWidget.registry.RegisterToasterWidget = publicWidget.Widget.extend({
     start: function () {
         const message = this.$el.data('message');
```
Comment (Polymorphe57) [inline addons/website_event/static/src/js/register_toaster_widget.js:21]: why remove markup?

**Comment by Polymorphe57** [comment]
robodoo rebase-ff r+

**Comment by Polymorphe57** [comment]
robodoo r+

**Pull request closed without merging**